anima-jws-voucher.mkd

```diff
@@ -189,20 +169,21 @@ The following figure provides an example of JSON Voucher Data:
 
 ## JWS Protected Header
 
-The JWS Protected Header defined in {{RFC7515}} uses the standard header parameters "alg", "typ", and "x5c".
-The "alg" parameter MUST contain the algorithm type used to create the signature, e.g., "ES256" as defined in {{Section 4.1.1 of RFC7515}}.
-If present, the "typ" parameter SHOULD contain the value "voucher-jws+json" as defined in {{Section 4.1.9 of RFC7515}}.
-If X.509 (PKIX) certificates {{RFC5280}} are used, the "x5c" parameter SHOULD contain the base64-encoded (not base64url-encoded) X.509 v3 (DER) certificate and chain as defined in {{Section 4.1.6 of RFC7515}}.
+The JWS Protected Header defined in {{RFC7515}} uses the standard header parameters `alg`, `typ`, and `x5c`:
+
+* The `alg` parameter MUST contain the algorithm type (e.g., `ES256`) used to create the signature as defined in {{Section 4.1.1 of RFC7515}}.
+* The `typ` parameter is optional and used when more than one kind of object could be present in an application data structure as described in {{Section 4.1.9 of RFC7515}}. If present, the `typ` parameter MUST contain the value `voucher-jws+json`.
```

Comment:
Inlined from RFC7515 why it is optional and when it should be included.
Also I think the value is a MUST, not a SHOULD -- or was there any case where the JWS Voucher format could put something else here?

Reply:
Agreed. In BRSKI-PRM we also treat it as mandatory.